src/coreComponents/physicsSolvers/wavePropagation/AcousticVTIWaveEquationSEM.cpp

```diff
@@ -0,0 +1,1027 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOS Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+
+/**
+ * @file AcousticVTIWaveEquationSEM.cpp
+ */
+
+#include "AcousticVTIWaveEquationSEM.hpp"
+#include "AcousticVTIWaveEquationSEMKernel.hpp"
+
+#include "finiteElement/FiniteElementDiscretization.hpp"
+#include "fieldSpecification/FieldSpecificationManager.hpp"
+#include "fieldSpecification/PerfectlyMatchedLayer.hpp"
+#include "mainInterface/ProblemManager.hpp"
+#include "mesh/ElementType.hpp"
+#include "mesh/mpiCommunications/CommunicationTools.hpp"
+#include "WaveSolverUtils.hpp"
+
+namespace geos
+{
+
+using namespace dataRepository;
+
+AcousticVTIWaveEquationSEM::AcousticVTIWaveEquationSEM( const std::string & name,
+                                                  Group * const parent ):
+  WaveSolverBase( name,
+                  parent )
+{
+
+  registerWrapper( viewKeyStruct::sourceNodeIdsString(), &m_sourceNodeIds ).
```

Comment:
You can remove every  registerWrapper except the last one (pressureAtReceivers), all othr wrappers are defined inside WaveSolverBase so they are not necessary here (you can take a look to the AcousticWaveEquationSEM.cpp file)

Reply:
Done :)